python/ray/tune/tune.py

```diff
@@ -846,14 +847,15 @@ class and registered trainables.
 
     progress_metrics = _detect_progress_metrics(_get_trainable(run_or_experiment))
 
-    # Create syncer callbacks
+    # Create default logging + syncer callbacks
     callbacks = _create_default_callbacks(
         callbacks,
         sync_config=sync_config,
         air_verbosity=air_verbosity,
         metric=metric,
         progress_metrics=progress_metrics,
     )
+    air_usage.tag_callbacks(callbacks)
```

Comment:
nit: Would it be beneficial to move this before `_create_default_callbacks`? Would we be able to derive more information about user-specified callbacks?

Reply:
I explicitly ignore default callbacks, but now that you're pointing it out, there's no need to do this if I just put it before we populate the default ones.